gson/src/test/java/com/google/gson/stream/JsonReaderTest.java

```diff
@@ -79,12 +82,11 @@ public void testEscapedNewlineNotAllowedInStrictMode() throws IOException {
     String json = "\"\\\n\"";
     JsonReader reader = new JsonReader(reader(json));
     reader.setStrictness(Strictness.STRICT);
-    try {
-      reader.nextString();
-      fail();
-    } catch (IOException expected) {
-      assertThat(expected.getMessage()).contains("Cannot escape a newline character in strict mode!");
-    }
+    IOException expected = assertThrows(IOException.class, () -> {
```

Comment:
It is cleaner to write `IOException expected = assertThrows(IOException.class, reader::nextString` IMO. intellij gives this as an automatic suggestino for me. This applies to all cases in this PR.

Reply:
Figured id stick with lambda since it was mentioned in the main pull request, but yes I agree that one looks cleaner, ill fix it